### ISSUE TITLE
Finish setting up this repo

### ISSUE THREAD

**Issue by n-a-t-e** [HakaiInstitute/hakai-datasets-upptime#272]
Somethings are broken in this repo:
 - there are datasets being reported as down that are actually up
 - the uptime status page is broken - https://hakaiinstitute.github.io/hakai-datasets-upptime/

Also there is no mechanism to auto update the list of datasets. I have a python script to generate the datasets list, this could probably be added to this repo to have it auto-update daily

**Comment by n-a-t-e** [comment]
This is the python script I used to generate the `sites` block in https://github.com/HakaiInstitute/hakai-datasets-upptime/blob/master/.upptimerc.yml:

```python
import pandas as pd

url='https://catalogue.hakai.org/erddap/tabledap/allDatasets.csv?datasetID&datasetID!="allDatasets"&accessible="public"'

sites=pd.read_csv(url)['datasetID']

for dataset_id in sites:
    print (f"""  - name: {dataset_id}\n    url: https://catalogue.hakai.org/erddap/tabledap/{dataset_id}.htmlTable?time&time<=now-1week&distinct()
    expectedStatusCodes:
      - 200
      - 201
      - 404""")
```

**Comment by n-a-t-e** [comment]
Committed this script see - https://github.com/HakaiInstitute/hakai-datasets-upptime/blob/master/generate-sites-yaml-block.py

**Comment by JessyBarrette** [comment]
I think I covered that issue with the lastest changes let  me know if you think of something else.

I guess I one more thing would maybe to rebuild the from readme page more frequentely since it seems out of sync with the issues raised and dropped.

I'm also wondering if we could host a webpage similar to https://cioos-siooc.github.io/cwatch-upptime/

**Comment by n-a-t-e** [comment]
Broken links should be fixed by b48729d
